app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include IdvStepConcern
+    include RenderConditionConcern
+
+    before_action :confirm_step_allowed
+
+    check_or_render_not_found -> { self.class.enabled? }
+
+    def show
+      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
+      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+    end
+
+    def update
+      result = Idv::HowToVerifyForm.new.submit(how_to_verify_form_params)
+
+      analytics.idv_doc_auth_how_to_verify_submitted(
+        **analytics_arguments.merge(result.to_h),
+      )
+
+      if result.success?
+        if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
+          redirect_to idv_hybrid_handoff_url
+        else
+          redirect_to idv_document_capture_url
```

Comment:
This needs to set idv_session.flow_path = 'standard'.  Document capture requires it.

Reply:
This will be completed in LG-11458